spyder/config/main.py

```diff
@@ -154,7 +154,7 @@
               'pylab/inline/resolution': 72,
               'pylab/inline/width': 6,
               'pylab/inline/height': 4,
-              'pylab/inline/bbox_inches': False,

```

Comment:
Please bump `CONF_VERSION` for this change to take effect. Just follow the instructions above it to see what number you need to set for it.